InstallGuide.md

```diff
@@ -0,0 +1,74 @@
+# FitNesse / HSAC installation guide
+
+This is a step by step guide to get HSAC up and running on your machine. This guide is using Windows 10 as an OS, the steps will be similar in another environment. This guide uses IntelliJ as an Integrated Development Environment as it will allow you to get most out of FitNesse and to manage the running and updating of FitNesse as well as version control on the test files you are creating.
+ 
+ ##Supporting tools setup
+ We need to install these tools:
+ * Git for version control
+ * Java JDK so we can build the FitNesse framework from code
+ * Intellij (including the FitNesse Plugin) to edit the test files
+ 
+#####Git
+ Go to <https://git-scm.com/downloads>, download and install Git using the default settings.
+#####Java JDK
+On <http://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html> select the correct JDK for windows, do not forget to click *Accept Licence* first. After download, install this one as well.
+#####IntelliJ
+Download IntelliJ from <https://www.jetbrains.com/idea/download/>, specifically choose the Community Edition. After download, install this as well using the default options.
+######Chrome
+While it can be assumed most people have this already, just to make you do too.
```

Comment:
Other products have a link to where they can be obtained. I would say include a link to chrome download also.